src/nimlsp.nim

```diff
@@ -23,6 +22,11 @@ const
 
 var nimpath = explicitSourcePath
 
+when system.hostOS == "windows":
+  const storage = getEnv("TEMP")
+else:
+  const storage = "/tmp/nimlsp"
+
```

Comment:
Any reason you didn't do this as:
``` Nim
storage = when system.hostOS == "windows":
  getEnv("TEMP")
else:
  "/tmp/nimlsp"
```

in the const block where it was?

Reply:
`/tmp/nimlsp` is wrong too. https://nim-lang.github.io/Nim/os.html#getTempDir should be used

Reply:
Well, in the documentation it literally says **Please do not use this**, so I figured I shouldn't.. Another thing is that system is implicitly imported, so you can just do:
``` Nim
storage = when hostOS == "windows":
  getEnv("TEMP")
else:
  "/tmp/nimlsp"
```

Reply:
well, this thing is ever worse, so might as well use the lesser of two evils.

Reply:
Eh fair enough. @itmuckel, can you change it to just use `getTempDir` instead?

Reply:
Yeah, sure. I extracted it from the other consts, because VSCode showed errors. 🙄 
I did it this way, because the docs say "Don't use getTempDir".

I also realize that on Windows no dedicated directory will be created, but everything will be stuffed directly into TEMP.
`storage = getTempDir() & "nimlsp"` should do the trick. I'll try it.

Reply:
`getTempDir() / "nimlsp"`